src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureFlagOptions.cs

```diff
@@ -19,5 +19,11 @@ public class FeatureFlagOptions
         /// The time after which the cached values of the feature flags expire.  Must be greater than or equal to 1 second.
         /// </summary>
         public TimeSpan CacheExpirationInterval { get; set; } = AzureAppConfigurationOptions.DefaultFeatureFlagsCacheExpirationInterval;
+
+        /// <summary>
+        /// Selectively load only those feature flags which begin with this prefix.
+        /// The characters asterisk (*), comma (,) and backslash (\) are reserved and must be escaped using a backslash (\).
```

Comment:
>The characters asterisk (*), comma (,) and backslash (\) are reserved and must be escaped using a backslash (\). [](start = 12, length = 112)

There isn't anything preventing the user from using these reserved characters unescaped in the prefix. We could add validation for this case and return an error message.

Reply:
This validation could be done using regex to to error out if unescaped reserved characters are present in the prefix. However, if these characters are present in key name, we get 400 bad request from server. So we already have server side validation in place. I am not sure if we should introduce the same logic in config provider considering these reserved characters could change in future. We have similar requirement for the key filter input of `Select`, but we don't do any client side validation there either.